### PR TITLE
SDIT-1370 Add ability to migrate incident reports

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -64,6 +64,7 @@ tasks {
       "buildActivityApiModel",
       "buildNomisSyncApiModel",
       "buildAdjudicationApiModel",
+      "buildIncidentsApiModel",
```
Comment (karenmillermoj) [inline build.gradle.kts:67]: I have added this in, even though there are minimal endpoints at the mo

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsMigrationFilter.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.incidents
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDate
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Filter specifying what should be migrated from NOMIS to the Non-Associations service")
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsMigrationFilter.kt:8]: incidents service?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsMigrationFilter.kt [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+data class IncidentsMigrationFilter(
+
+  @Schema(
+    description = "Only include non-associations created on or after this date",
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsMigrationFilter.kt:12]: incidents?

**Review by petergphillips — APPROVED** [review]
LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsMigrationFilter.kt (hunk 2 above)
Comment (karenmillermoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsMigrationFilter.kt:8]: good spot

**Pull request merged**